### ISSUE TITLE
Minor bug in Invoke-DellBIOSUpdate

### ISSUE THREAD

**Issue by Ginolard** [SCConfigMgr/ConfigMgr#137]
Another empty string problem ;)

Sensibly, you replace the $Password variable with "Password Removed" in the log output.  However, if $password is empty because there's no BIOS password the output looks like this.

```
Using the following switches for Flash64W.exe: <password removed>/<password removed>b<password removed>=<password removed>D<password removed>:<password removed>\<password removed>s<password removed>o<password removed>u<password removed>r<password removed>c<p
assword removed>e<password removed>s<password removed>\<password removed>D<password removed>r<password removed>i<password removed>v<password removed>e<password removed>r<password removed>s<password removed>\<password removed>P<password removed>a<password rem
oved>c<password removed>k<password removed>a<password removed>g<password removed>e<password removed>s<password removed>\<password removed>D<password removed>e<password removed>l<password removed>l<password removed>\<password removed>L<password removed>a<pass
word removed>t<password removed>i<password removed>t<password removed>u<password removed>d<password removed>e<password removed> <password removed>5<password removed>2<password removed>5<password removed>0<password removed>\<password removed>B<password remove
d>I<password removed>O<password removed>S<password removed>\<password removed>A<password removed>2<password removed>0<password removed>\<password removed>5<password removed>2<password removed>5<password removed>0<password removed>A<password removed>2<passwor
d removed>0<password removed>.<password removed>e<password removed>x<password removed>e<password removed> <password removed>/<password removed>s<password removed> <password removed>/<password removed>l<password removed>=<password removed>C<password removed>:
<password removed>\<password removed>t<password removed>e<password removed>m<password removed>p<password removed>\<password removed>l<password removed>o<password removed>g<password removed>.<password removed>l<password removed>o<password removed>g<password r
emoved>
```

Not a big deal as it doesn't stop the BIOS being updated - just annoying to read ;)

**Comment by NickolajA** [comment]
Should be fixed in 1.1.0, thank you for reporting this.

**Comment by Ginolard** [comment]
No problem.  Glad to help support this project!  It's been of huge benefit
to us
